docs/specs/code/hardware-wallets/ledger/README.md

```diff
@@ -208,6 +208,76 @@ app/components => View base components
 
 # Iteration-2
 
+## Proposal
+Updated UI to for Ledger Nano X support using USB cable.
+
+## Related PRs
+Implementation: https://github.com/Emurgo/yoroi-frontend/pull/660
+
+# Iteration-3
+
+## Proposal
+As in Windows 10, version 1903 (10th May 2019 update) native support for U2F has been added and it's affecting current Yoroi UI.<br>
+Demo can be found here: https://youtu.be/Hlo6wOA16MA
+
+In order fix UI problem, support for [WebAuthn Transport](https://www.npmjs.com/package/@ledgerhq/hw-transport-webusb) is proposed.
+
+As WebAuthn Transport supports all [targetted browsers](https://caniuse.com/#search=Web%20Authentication%20API) it will be used as the default protocol. U2F transport will still be available.
+
+## Module structure
+Till now we have two modules and one repository at: https://github.com/Emurgo/yoroi-extension-ledger-bridge<br>
+`master` branch acts as connector module<br>
+`gh-pages` branch is the actual `iframe` which communicates with the Ledger device<br>
+making one repository per module feels more better now and it will help proper testing of new URL before release.<br>
+Hence proposing two new repository like:<br>
+1. `yoroi-extension-ledger-connect-handler` as handler of opening, closing and communication with `yoroi-extension-ledger-connect-website`
+2. `yoroi-extension-ledger-connect-website` as website which will communicate with the Ledger device
+
+Also proposing to change hosting from Github pages to **Netlify**, because debugging and development is bit difficult with Github pages.
+With **Netlify** we can set `mainnet` for **Production Netlify** and for other networks we can use **Development Netlify**.
```

Comment:
I think this makes sense for PRs so you can  easily test your PR on a temporary URL.
We may want to keep the main URL on Github though so it's clear the code on Github is deployed directly without modifications (otherwise if somebody is able to hijack the DNS for yoroi-wallet then it's a security problem)